src/index.js

```diff
@@ -0,0 +1,59 @@
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
+import './main.scss';
+import domManager from './js/info';
+
+const APIKEY = 'eaed0043eb661ba7a7f56160196f9ad9';
+const btn = document.querySelector('.search__btn');
+const search = document.getElementById('search__input');
+const error = document.querySelector('.error');
+
+async function getWeather(location, unit) {
+  const container = document.querySelector('.info');
+  container.innerHTML = '';
+  try {
+    const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${location}&APPID=${APIKEY}&units=${unit}`, { mode: 'cors' });
+    const weatherData = await response.json();
+    domManager.setBackground(weatherData.weather[0].main);
+    const infoElements = domManager.setInfo(weatherData);
+    container.innerHTML += infoElements;
+    const icon = document.createElement('img');
+    const iconContainer = document.querySelector('.info__bottom-right');
+    icon.src = `http://openweathermap.org/img/wn/${weatherData.weather[0].icon}@2x.png`;
+    iconContainer.appendChild(icon);
+  } catch (e) {
+    error.innerHTML = 'You must enter a valid city';
+    error.classList = 'error active';
+  }
+}
+
+function searchLocation() {
```

Comment:
You ought to use Es6 syntax as can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)

Like:
`const searchLocation = () => {`